### ISSUE TITLE
Know vulnerabilities is None

### ISSUE THREAD

**Issue by KongLynn** [fkie-cad/cwe_checker#257]
I have scanned multiple firmwares but none of their known vulnerabilities. Can you tell me the specific scan contents of the known vulnerabilities module？ Thank you very much.

**Comment by Enkelmann** [comment]
Do you mean the known vulnerabilities module of FACT? If yes, then this is the wrong repository. The cwe_checker outputs CWE warnings and not known vulnerabilities.

**Comment by KongLynn** [comment]
Yes I mean the known vulnerabilities module of FACT.Do you mean there is no know vulnerabilities check in FACT？
